litmus-portal/graphql-server/pkg/k8s/cluster.go

```diff
@@ -153,7 +151,7 @@ func GetServerEndpoint() (string, error) {
 			Scheme = "http"
 		}
 
-		FinalUrl = Scheme + "://" + IPAddress + "/" + IngressPath
+		FinalUrl = Scheme + "://" + wrapIPV6(IPAddress) + "/" + IngressPath
```

Comment:
Should we give an option between IPV6 and IPV4 by a flag?

Reply:
I can see a function, will it automatically detect?

Reply:
Yes auto detect